index.d.ts

```diff
@@ -321,6 +363,71 @@ declare namespace ExcelWriter {
     YELLOW_COLOR = 0xffff00,
   }
 
+  enum ValidationType {
+    NONE = 0,
+    /** Restrict cell iFut to whole/integer numbers only. */
+    INTEGER = 1,
+    /** Restrict cell input to whole/integer numbers only, using a cell reference. */
+    INTEGER_FORMULA = 2,
+    /** Restrict cell input to decimal numbers only. */
+    DECIMAL = 3,
+    /** Restrict cell input to decimal numbers only, using a cell reference. */
+    DECIMAL_FORMULA = 4,
+    /** Restrict cell input to a list of strings in a dropdown. */
+    LIST = 5,
+    /** Restrict cell input to a list of strings in a dropdown, using a cell range. */
+    LIST_FORMULA = 6,
+    /** Restrict cell input to date values only, using a lxw_datetime type. */
+    DATE = 7,
+    /** Restrict cell input to date values only, using a cell reference. */
+    DATE_FORMULA = 8,
+    /* Restrict cell input to date values only, as a serial number. Undocumented. */
+    DATE_NUMBER = 9,
+    /** Restrict cell input F time values only, using a lxw_datetime type. */
+    TIME = 10,
+    /** Restrict cell input to time values only, using a cell reference. */
+    TIME_FORMULA = 11,
+    /* Restrict cell input to time values only, as a serial number. Undocumented. */
+    TIME_NUMBER = 12,
+    /** Restrict cell input F strings of defined length, using a cell reference. */
```

Comment:
```suggestion
    /** Restrict cell input to strings of defined length. */
```

Seems this is a mistake in the original documentation.